### ISSUE TITLE
1.16.5 Crash when trying to use the blood spell parchment and bind it to the blood gem

### ISSUE THREAD

**Issue by Pyro206** [MintTeaTx/Tome-of-Blood#13]
When using a blood gem in the inscription table and trying to bind a spell to it using the blood spell paper it causes a server tick loop crashing my game this happened on all the mods 6 so im not sure if its ur issue or not
[crash-2021-12-25_23.13.03-server.txt](https://github.com/MintTeaTx/Tome-of-Blood/files/7776547/crash-2021-12-25_23.13.03-server.txt)

**Comment by Pyro206** [comment]
The game crashed whilst exception in server tick loop
Error: java.lang.NoSuchMethodError: com.hollingsworth.arsnouveau.api.util.SpellRecipeUtil.getSpellsFromTagString(Ljava/lang/String;)Ljava/util/List;
Exit Code: -1

**Comment by Arenuphis** [comment]
Yeah I had the same issue on a home made modpack based on Create:above and beyond, when trying to figure out how the blood parchment and blood gem worked
Happened both on a server and on a single player world
One instance was shift right clicking with one on the other in an enscription table, the other was when right clicking with one while holding the other in my of-hand

Information about how to use those two is pretty sparse and vague to begin with tbh
